src/utils/tablify.js

```diff
@@ -54,12 +55,17 @@ const reduceKeys = (keys, row) => {
 	return keys.concat(newKeys);
 };
 
-const getKeys = data =>
-	Object.entries(data)
+const getKeys = (data, limit = 3, loop = 1) => {
+	if (loop > limit) {
+		throw new PrintError(
```

Comment:
This will crash Lisk Commander rather than printing the message in table format. Is that what we want?

Reply:
It doesn't show in table format, but it doesn't crash the commander neither.
I think it's more consistent with other types of `Error`